javascript-challenges.test.js

```diff
@@ -14,30 +16,167 @@
 
 // a) Create a test with expect statements using the variables provided.
 
-const secretCodeWord1 = "Lackadaisical"
-// Expected output: "L4ck4d41s1c4l"
-const secretCodeWord2 = "Gobbledygook"
-// Expected output: "G0bbl3dyg00k"
-const secretCodeWord3 = "Eccentric"
-// Expected output: "3cc3ntr1c"
+
+// describe("vowelToNumber", () => {
+//     it("takes in a string and returns a string with a coded message", () => {
+//         const secretCodeWord1 = "Lackadaisical"
+//         expect(vowelToNumber(secretCodeWord1)).toEqual("Lackadaisical")
+//         // Expected output: "L4ck4d41s1c4l"
+        
+//         const secretCodeWord2 = "Gobbledygook"
+//         expect(vowelToNumber(secretCodeWord2)).toEqual("Gobbledygook")
+//         // Expected output: "G0bbl3dyg00k"
+
+//         const secretCodeWord3 = "Eccentric"
+//         expect(vowelToNumber(secretCodeWord3)).toEqual("Eccentric")
+//         // Expected output: "3cc3ntr1c"
+//     })
+// })
+
+
+// OUTPUT: ReferenceError: vowelToNumber is not defined
+
 
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
 
+//INPUT: a string
+//OUTPUT: a returned string that converts "a" to 4, "e" to 3, "i" to 1, and "o" to 0.
+
+// const vowelToNumber = .toLowerCase(string) => {
+//     let codeCreater = "";
+
+//     for (let i = 0; i < string.length; i++) {
+//         let charCode = string[i];
+
+//         switch (charCode) {
+//             case 'a':
+//                 codeCreater += '4';
+//                 break;
+//             case 'e':
+//                 codeCreater += '3';
+//                  break;
+//             case 'i':
+//                 codeCreater += '1';
+//                  break;
+//             case 'o':
+//                 codeCreater += '0';
+//                 break;
+//             default:
+//                 codeCreater += charCode;
+//         }
+//     }
+//     // .toLowerCase().split('')
+//     // .filter('a' = '4', 'e' = '3', 'i' = '1', 'o'= '0')
+//     // .join(' ')
+//     return codeCreater
+// }
+
+// const secretCodeWord1 = "Lackadaisical"
+// const secretCodeWord2 = "Gobbledygook"
+// const secretCodeWord3 = "Eccentric"
+
+// console.log(vowelToNumber(secretCodeWord1))
+// console.log(voweltoNumber(secretCodeWord2))
+// console.log(vowelToNumber(secretCodeWord3))
+
+// create function that holds substitute letter value and iterates through string
+//2nd option
+
```

Comment:
You're lacking pseudocode for this attempt, but I think I can figure out what you're trying to do.  Remember to stick with what you know first.  I think you got confused with the switch statements.  Keep going Ant!  You'll get it!